### ISSUE TITLE
Running create schema queries not working on fresh install

### ISSUE THREAD

**Issue by jamesnvc** [terminusdb/terminusdb#324]
**Describe the bug**

After building TerminusDB and trying to run queries from tutorials in the web interface, I get an error stating `Badly formed ast after compilation with term`.

**To Reproduce**
1. Open the web interface at localhost:6363
2. Create a new database
3. Go to "Query"
4. Run, e.g. `WOQL.doctype("scm:student").property("scm:name", "xsd:string")` (although various other queries from the tutorials have given me the same error)
5. Error message like `Badly formed ast after compilation`

<details><summary>Full error message</summary>

```
Badly formed ast after compilation with term: insert('terminusdb:///schema#student','http://www.w3.org/1999/02/22-rdf-syntax-ns#type','http://www.w3.org/2002/07/owl#Class',"schema/main"),insert('terminusdb:///schema#student','http://www.w3.org/2000/01/rdf-schema#subClassOf','http://terminusdb.com/schema/system#Document',"schema/main"),insert('terminusdb:///schema#name','http://www.w3.org/1999/02/22-rdf-syntax-ns#type','http://www.w3.org/2002/07/owl#DatatypeProperty',"schema/main"),insert('terminusdb:///schema#name','http://www.w3.org/2000/01/rdf-schema#range','http://www.w3.org/2001/XMLSchema#string',"schema/main"),insert('terminusdb:///schema#name','http://www.w3.org/2000/01/rdf-schema#domain','terminusdb:///schema#student',"schema/main")less

{
  "data": {
    "@type": "api:WoqlErrorResponse",
    "api:error": {
      "@type": "api:WOQLSyntaxError",
      "api:error_term": "insert('terminusdb:///schema#student','http://www.w3.org/1999/02/22-rdf-syntax-ns#type','http://www.w3.org/2002/07/owl#Class',\"schema/main\"),insert('terminusdb:///schema#student','http://www.w3.org/2000/01/rdf-schema#subClassOf','http://terminusdb.com/schema/system#Document',\"schema/main\"),insert('terminusdb:///schema#name','http://www.w3.org/1999/02/22-rdf-syntax-ns#type','http://www.w3.org/2002/07/owl#DatatypeProperty',\"schema/main\"),insert('terminusdb:///schema#name','http://www.w3.org/2000/01/rdf-schema#range','http://www.w3.org/2001/XMLSchema#string',\"schema/main\"),insert('terminusdb:///schema#name','http://www.w3.org/2000/01/rdf-schema#domain','terminusdb:///schema#student',\"schema/main\")"
    },
    "api:message": "Badly formed ast after compilation with term: insert('terminusdb:///schema#student','http://www.w3.org/1999/02/22-rdf-syntax-ns#type','http://www.w3.org/2002/07/owl#Class',\"schema/main\"),insert('terminusdb:///schema#student','http://www.w3.org/2000/01/rdf-schema#subClassOf','http://terminusdb.com/schema/system#Document',\"schema/main\"),insert('terminusdb:///schema#name','http://www.w3.org/1999/02/22-rdf-syntax-ns#type','http://www.w3.org/2002/07/owl#DatatypeProperty',\"schema/main\"),insert('terminusdb:///schema#name','http://www.w3.org/2000/01/rdf-schema#range','http://www.w3.org/2001/XMLSchema#string',\"schema/main\"),insert('terminusdb:///schema#name','http://www.w3.org/2000/01/rdf-schema#domain','terminusdb:///schema#student',\"schema/main\")",
    "api:status": "api:failure",
    "end": "2021-05-17T18:54:16.812Z",
    "start": "2021-05-17T18:54:16.812Z",
    "duration": "2021-05-17T18:54:16.812Z"
  }
}
```
</details>

**Expected behavior**
The quads would be inserted

**Screenshots**

![screenshot-2021-05-17_15-03-40](https://user-images.githubusercontent.com/38405/118542504-18095280-b721-11eb-84bd-ec099bf083f8.png)

**Info (please complete the following information):**
 - OS: Ubuntu 20.04.2 LTS
 - How did you run terminus-server: manual installation
- SWI Prolog version: 8.3.21 (via `swivm`)

**Comment by jamesnvc** [comment]
I also get the same result using the python client:

```python
client = WOQLClient(server_url='https://127.0.0.1:6363')
client.connect(key='foobarbaz', account='admin', user='admin', db='university')
WOQLQuery().doctype("scm:student").property("scm:name", "xsd:string").execute(client, "student schema created.")
```
throws the same error:

```
terminusdb_client.errors.DatabaseError: Badly formed ast after compilation with term: insert('terminusdb:///schema#student','http://www.w3.org/1999/02/22-rdf-syntax-ns#type','http://www.w3.org/2002/07/owl#Class',"schema/main"),insert('terminusdb:///schema#student','http://www.w3.org/2000/01/rdf-schema#subClassOf','http://terminusdb.com/schema/system#Document',"schema/main"),insert('terminusdb:///schema#name','http://www.w3.org/1999/02/22-rdf-syntax-ns#type','http://www.w3.org/2002/07/owl#DatatypeProperty',"schema/main"),insert('terminusdb:///schema#name','http://www.w3.org/2000/01/rdf-schema#range','http://www.w3.org/2001/XMLSchema#string',"schema/main"),insert('terminusdb:///schema#name','http://www.w3.org/2000/01/rdf-schema#domain','terminusdb:///schema#student',"schema/main")
```

**Comment by jamesnvc** [comment]
After running the query inside swipl, I was able to see that the error was happening because the `schema/main` graph didn't exist. I was able to manually create the graph and then things work.

1. Should that graph have been automatically created at some point?

2. It was very difficult to figure that out via the error messages, because it seems the errors in `woql_compile:compile_query/4` somehow get swallowed and just sent as a generic `badly_formed_ast` term.
